ReactAndroid/src/main/java/com/facebook/react/modules/image/ImageLoaderModule.java

```diff
@@ -81,22 +82,23 @@ public void getSize(
     Uri uri = Uri.parse(uriString);
     ImageRequest request = ImageRequestBuilder.newBuilderWithSource(uri).build();
 
-    DataSource<CloseableReference<CloseableImage>> dataSource =
-      Fresco.getImagePipeline().fetchDecodedImage(request, mCallerContext);
+    DataSource<CloseableReference<PooledByteBuffer>> dataSource =
+      Fresco.getImagePipeline().fetchEncodedImage(request, mCallerContext);
 
-    DataSubscriber<CloseableReference<CloseableImage>> dataSubscriber =
-      new BaseDataSubscriber<CloseableReference<CloseableImage>>() {
+    DataSubscriber<CloseableReference<PooledByteBuffer>> dataSubscriber =
+      new BaseDataSubscriber<CloseableReference<PooledByteBuffer>>() {
         @Override
         protected void onNewResultImpl(
-            DataSource<CloseableReference<CloseableImage>> dataSource) {
+            DataSource<CloseableReference<PooledByteBuffer>> dataSource) {
           if (!dataSource.isFinished()) {
             return;
           }
-          CloseableReference<CloseableImage> ref = dataSource.getResult();
+          CloseableReference<PooledByteBuffer> ref = dataSource.getResult();
           if (ref != null) {
+            EncodedImage image = ref.get();
```

Comment:
Hmm, how come this doesn't warn about trying to assign a `PooledByteBuffer` to an `EncodedImage`? Shouldn't `ref` and `dataSource` use `EncodedImage` as the type arg?